quantumblacklabs/kedro-viz
# [KED-2966] Replace the QB favicon with the Kedro one

## Description

Resolves [KED-2966](https://jira.quantumblack.com/browse/KED-2966).

## QA notes

[Check it here](http://kedro-viz-fe.s3-website.eu-west-2.amazonaws.com/feature/update-favicon-to-kedro-icon%23ked-2966/?data=spaceflights).

## Checklist


## Legal notice


- I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
- I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
- I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
